src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -48,18 +48,27 @@ function errorMsg(msg, error) {
 }
 
 
-startButton.addEventListener('click', () => {
-  const options = {audio: true, video: true};
-  const displaySurface = preferredDisplaySurface.options[preferredDisplaySurface.selectedIndex].value;
-  if (displaySurface !== 'default') {
-    options.video = {displaySurface};
+startStopButton.addEventListener('click', () => {
+  if (startStopButton.textContent === 'Start') {
```

Comment:
textContent  is a state of dom. Wouldn't it be better make this decision based on explicitly defined boolean variable?

Reply:
💯 
added a `displayMediaStarted` boolean